backend/src/common/packages/packages.ts

```diff
@@ -0,0 +1,4 @@
+export { config } from './config/config.js';
+export { database } from './database/database.js';
+export { logger } from './logger/logger.js';
+export { token } from './token/token.js';
```

Comment:
add controller

Reply:
But controller does not exports instance like other 4 packages. Here is what controller exports from controller.ts:
```
export { ControllerBase } from './controller-base.package.js';
export { ControllerHooks } from './enums/enum.js';
export {
    type ApiHandlerOptions,
    type ApiHandlerResponse,
} from './types/types.js';
```
Should the packages.ts file look like this:
```
export { config } from './config/config.js';
export { database } from './database/database.js';
export { logger } from './logger/logger.js';
export { token } from './token/token.js';
export {ControllerBase, ControllerHooks} from './controller/controller.js'
```
?